### ISSUE TITLE
Help for running 

### ISSUE THREAD

**Issue by Rahiti-cell** [scribble-rs/scribble.rs#85]
Hello,

Your work is fantastic.

Unfortunately, even following your instructions, I cannot manage to run the site on a local network.
I use a mac.

I am a beginner in this field.

I would like to tell you that by playing on the website, by using customs words, only the word used is changed. This helps players and therefore limits the choice to one word.

Well done again

**Comment by Bios-Marcel** [comment]
What exactly is the problem?

Also, if you only want custom words to be used, move the chance slider to the very right.

**Comment by Rahiti-cell** [comment]
For me : 

I would like to have the detail of the procedure to follow to run the site in local network on my mac.


On the Website :

For my second comment, I have dragged the cursor to the right. On the other hand, among the proposed words, only the one chosen is modified for the next player. Example: Player 1 (word list: Plane / Boat / Car). Player chooses "Car". When player 2 chooses his word, the list is: Plane / Boat / Bus. Only the latter has been changed.

Thank you for your speed.
